### ISSUE TITLE
Cannot find vtt.py file

### ISSUE THREAD

**Issue by lindsaywu** [aws-samples/aws-transcribe-captioning-tools#8]
Thank you for sharing the code. But I cannot find vtt.py file to convert json to vtt file. Any idea?

**Comment by eggoynes** [comment]
You are right, I don't see a vtt.py either. Let me look around and see if we have it in an internal repo.
